source/create_my_testbed.rst

```diff
@@ -28,14 +28,14 @@ Parameter file can be read using ``Asterix.fits_functions.read_parameter_file``
     config = useful.read_parameter_file(parameter_file)
     modelconfig = config["modelconfig"]
 
-    generic_os = OptSy.Optical_System(modelconfig)
+    generic_os = OptSy.OpticalSystem(modelconfig)
 
 
 For the moment, this ``generic_os`` does not do anything, it's like an empty pupil plane. 
 
-Each ``Optical_System`` has an attribute function ``EF_through`` which describes the effect this Optical_System has 
-on the electrical field when going through it. Once this function is defined, all Optical_System have access to 
-a library of functions, common to all ``Optical_System`` : ``todetector`` (electrical field in the next focal plane), 
+Each ``OpticalSystem`` has an attribute function ``EF_through`` which describes the effect this Optical System has
```

Comment:
No space

Reply:
Are you sure? I actually think it should just be "optical system" (lower case) since it talks about the actual optical system you are simulating rather than the class object `OpticalSystem`. Thoguhts?

Reply:
yeah i see your point. ok we can leave it like that

Reply:
Ok. I still pushed the change to lower-case letters.